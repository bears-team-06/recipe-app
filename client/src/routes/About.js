import React from "react";
const About = () => {
  return (
    <div className={"about"}>
      <div className="col-sm-5 text-center container" id="about-content">
        <p>
          This project is a collaborative effort from&nbsp;
          <a href="https://github.com/bears-team-06">Bears Team 6</a> of Chingu
          Beta Voyage.
        </p>
        <p>
          Visit <a href="https://chingu.io/">Chingu Cohorts website</a> and join
          a supportive group of life-long learners.
        </p>
      </div>
    </div>
  );
};

export default About;
