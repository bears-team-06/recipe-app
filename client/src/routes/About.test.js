import {shallow} from 'enzyme'
import React from 'react';
import About from "./About";

it('with text and is disabled true', () => {
    expect(shallow(<About />)).toMatchSnapshot();
})