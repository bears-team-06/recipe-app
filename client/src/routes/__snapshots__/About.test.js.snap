// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`with text and is disabled true 1`] = `
<div
  className="about"
>
  <div
    className="col-sm-5 text-center container"
    id="about-content"
  >
    <p>
      This project is a collaborative effort fromÂ 
      <a
        href="https://github.com/bears-team-06"
      >
        Bears Team 6
      </a>
       of Chingu Beta Voyage.
    </p>
    <p>
      Visit 
      <a
        href="https://chingu.io/"
      >
        Chingu Cohorts website
      </a>
       and join a supportive group of life-long learners.
    </p>
  </div>
</div>
`;
